config/cucumber.yml

```diff
@@ -2,7 +2,13 @@
 rerun = File.file?('rerun.txt') ? IO.read('rerun.txt') : ""
 rerun_opts = rerun.to_s.strip.empty? ? "--format #{ENV['CUCUMBER_FORMAT'] || 'progress'} features" : "--format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} #{rerun}"
 std_opts = "-r features --format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} --strict --tags ~@wip"
+std_opts_aa = "-r features/aa -r features/support --format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} --strict --tags ~@wip_aa"
+std_opts_hq = "-r features/hq -r features/support --format #{ENV['CUCUMBER_FORMAT'] || 'pretty'} --strict --tags ~@wip_hq"
 %>
```

Comment:
@PurityControl  I checked that only the specified spec definition files are loaded for aa and for hq.
Also, here is where we specify what folders should be loaded when `cucumber -p aa` or cucumber -p hq` is called.
Only the `support` folder is shared and that is loaded first by cucumber.